shell/platform/darwin/ios/framework/Source/accessibility_text_entry.mm

```diff
@@ -241,22 +237,31 @@ - (BOOL)accessibilityActivate {
 }
 
 - (NSString*)accessibilityLabel {
+  NSString* label = [super accessibilityLabel];
+  if (label != nil)
+    return label;
   return [self textInputSurrogate].accessibilityLabel;
 }
 
 - (NSString*)accessibilityHint {
+  NSString* hint = [super accessibilityHint];
+  if (hint != nil)
+    return hint;
   return [self textInputSurrogate].accessibilityHint;
 }
 
 - (NSString*)accessibilityValue {
+  NSString* value = [super accessibilityValue];
```

Comment:
Could we have an issue with asynchrony here? `[super accessibilityValue]` would return the Dart-side value that's not updated instantly.